int/hwsc-file-transaction-svc/hwsc-file-transaction-svc.js

```diff
@@ -13,7 +13,7 @@ const hwscFileTransactionSvcProtoPkgDef = protoLoader.loadSync('', options);
 const hwscFileTransactionSvcPbJs = grpc.loadPackageDefinition(hwscFileTransactionSvcProtoPkgDef)
```

Comment:
as of last night you are missing a directory in the `options`, check user-svc  and document-svc